terraform/bambictf.tf

```diff
@@ -196,6 +204,34 @@ systemctl start wg-quick@internal
 TERRAFORMEOF
 }
 
+resource "hcloud_server" "moloch" {
+  name        = "moloch${count.index + 1}"

```

Comment:
We can drop the count here unless we plan having more than one moloch (which I don't think)

Reply:
ok